### PR TITLE
67 time series

### LINKED ISSUE #67: Time Series: show event specific time data
When we click on an event we need to redraw the time series graph to show the posts for that event limited to the time window of the event.

### PR DESCRIPTION
**Pull request by justinlueders**
This may or may not work...I did all sorts of git stuff so who knows.

#Fix #67

### CODE REVIEW

File: tasks/workers/etl/sandbox/etl.js [hunk 1]
```diff
@@ -64,14 +64,54 @@ function getUrlFromNodeId(node){
...
+function buildTimeseries(nodes){
+  var dateMap = {};
+  var firstDate;
+  for(var i =0; i< nodes.length; i++){
+    var node = nodes[i];
+    var date = new Date(node.time * 1000);
+    if(!firstDate || node.time < firstDate){
+      firstDate = node.time;
+    }
+    var dateToHour =  new Date(date.getFullYear(), date.getMonth(), date.getDate(), date.getHours());
```
Comment (lukewendling) [inline tasks/workers/etl/sandbox/etl.js]: @justinlueders moment.js (which we have in the project) helps a lot with date-time conversion tasks like this. fyi... less hackery with native js Date methods. :clock1:

Comment (justinlueders) [inline tasks/workers/etl/sandbox/etl.js]: yeah I didn't know moment so it was a get to demoable state first issue...there are some other things in there I want to fix as well...like multiple ways to display data and how data is passed around...messages vs scope...
It's not done by any means.

**Comment by lukewendling** [comment]
moved to #72 to fix merge conflicts

File: tasks/workers/etl/sandbox/etl.js (hunk 1 above)
Comment (lukewendling) [inline tasks/workers/etl/sandbox/etl.js]: @justinlueders :+1:

**Pull request closed without merging**